src/components/common/RegAddEditOrgPerson.vue

```diff
@@ -180,40 +181,115 @@
 
             <!-- Roles -->
             <article class="mt-8">
-              <label>Roles</label>
-              <v-card flat rounded="sm" class="gray-card mt-4 px-4">
-                <v-row class="align-center">
-                  <v-col cols="4" v-if="showCompletingPartyRole" class="py-0">
-                    <v-checkbox
-                      class="mt-5"
-                      v-model="selectedRoles"
-                      :value="RoleTypes.COMPLETING_PARTY"
-                      :label="RoleTypes.COMPLETING_PARTY"
-                      :disabled="true"
-                    />
-                  </v-col>
-
-                  <v-col cols="4" v-if="showProprietorRole" class="py-0">
-                    <v-checkbox
-                      class="mt-5"
-                      v-model="selectedRoles"
-                      :value="RoleTypes.PROPRIETOR"
-                      :label="RoleTypes.PROPRIETOR"
-                      :disabled="true"
-                    />
-                  </v-col>
-
-                  <v-col cols="4" v-if="showPartnerRole" class="py-0">
-                    <v-checkbox
-                      class="mt-5"
-                      v-model="selectedRoles"
-                      :value="RoleTypes.PARTNER"
-                      :label="RoleTypes.PARTNER"
-                      :disabled="true"
-                    />
-                  </v-col>
-                </v-row>
-              </v-card>
+              <label>{{ getPeopleAndRolesResource.rolesTitle || 'Roles' }}</label>
```

Comment:
These are basically all the changes that were in the "IA" people-roles component.